_utilities/layout-grid.md

```diff
@@ -504,116 +357,150 @@ Add `grid-gap` to a grid row to add a gap (or gutter) between each column in the
+          </span>
+        </th>
+        <td data-title="Mixin" class="display-inline-flex flex-2">
+          <span>
+            grid-row
+          </span>
+        </td>
+        <td data-title="Accepted values" class="display-inline-flex flex-1">
+          <span>
+            -
+          </span>
+        </td>
+        <td data-title="Example" class="display-inline-flex flex-2">
+          <span>
+            @include grid-row
+          </span>
+        </td>
+      </tr>
+      <tr>
+        <th scope="row" data-title="Utility" class="tablet:maxw-card-lg display-inline-flex flex-2">
+          <span class="text-normal">
+            .grid-col <br />
+            .grid-col-<code>column count</code>
+          </span>
+        </th>
+        <td data-title="Mixin" class="display-inline-flex flex-2">
+          <span>
+            grid-col<code>(column count)</code>
+          </span>
+        </td>
+        <td data-title="Accepted values" class="display-inline-flex flex-1">
+          <span>
+            sm, md, lg, 2px, 05, 1, 3, 4, 6
+          </span>
+        </td>
+        <td data-title="Example" class="display-inline-flex flex-2">
+          <span>
+            @include grid-col <br/>
+            @include grid-col(6) <br/>
+            @include grid-col("auto")
+          </span>
+        </td>
+      </tr>
+      <tr>
+        <th scope="row" data-title="Utility" class="tablet:maxw-card-lg display-inline-flex flex-2">
+          <span class="text-normal">
+            .grid-gap <br />
+            .grid-gap-<code>gap size</code>
+          </span>
+        </th>
+        <td data-title="Mixin" class="display-inline-flex flex-2">
+          <span>
+            grid-gap(<a href="{{ site.baseurl }}/design-tokens/spacing-units/" class="token">units</a>)
+          </span>
+        </td>
+        <td data-title="Accepted values" class="display-inline-flex flex-1">
+          <span>
+            auto, fill, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12
+          </span>
+        </td>
+        <td data-title="Example" class="display-inline-flex flex-2">
+          <span>
+            @include grid-gap <br />
+            @include grid-gap("lg")
+          </span>
+        </td>
+      </tr>
+      <tr>
+        <th scope="row" data-title="Utility" class="tablet:maxw-card-lg display-inline-flex flex-2">
+          <span class="text-normal">
+            .grid-offset-<code>column-count</code>
+          </span>
+        </th>
+        <td data-title="Mixin" class="display-inline-flex flex-2">
+          <span>
+            grid-offset<code>(column-count)</code>
+          </span>
+        </td>
+        <td data-title="Accepted values" class="display-inline-flex flex-1">
+          <span>
+            none, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11
+          </span>
+        </td>
+        <td data-title="Example" class="display-inline-flex flex-2">
+          <span>
+            @include grid-offset(4)
+          </span>
+        </td>
+      </tr>
```

Comment:
**issue**: Order seems to be missing.

From live example:
```
// $order can be first, last, initial, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11
@include u-order($order);
```

![image](https://github.com/user-attachments/assets/f26dbcc8-e0f5-42ab-90dd-a381704b3f0b)

Reply:
I added this in [3102d17](https://github.com/uswds/uswds-site/pull/2276/commits/3102d174fefed7c70e8491b8a689ad3e269d4037)

But I'm curious if it's necessary or fits here. Order isn't exclusive to layout grid and we offer the [Order design token guidance](https://designsystem.digital.gov/design-tokens/order/). I'm also concerned about potential accessibility concerns given the changes we've made recently to remove visual order changes via CSS properties.

Do you think it's necessary to include here?

Reply:
Not if it's in a place that makes more sense and the information isn't lost. Thanks.

Reply:
Since the guidance is available on the order design token page, and since reordering via CSS generally goes against accessibility recommendations, I went ahead and removed the order utility mixin from this table.

Resolved in [c892bf4](https://github.com/uswds/uswds-site/pull/2276/commits/c892bf454ffbb136fab1f4fa1fa493265109aff9)